build/ci.go

```diff
@@ -1069,3 +1050,20 @@ func xgoTool(args []string) *exec.Cmd {
 	}
 	return cmd
 }
+
+func installLinter(lintPath string) {
+	lintInstaller := "golangci-lint-install.sh"
+	curlArgs := []string{
+		"-sSfL",
+		"-o" + lintInstaller,
+		"https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh",
+	}
+	build.MustRunCommand("curl", curlArgs...)
+
+	shArgs := []string{
+		lintInstaller,
+		"-b" + lintPath,
+	}
```

Comment:
Do we need to fix the version like the dockerfile?

Reply:
Thanks. I specified the same version of the binary installed in CI. 
Also, I refined the installation code a little bit.